### PR TITLE
chore: deploy sepolia contracts and change l2 chain id

### PR DESCRIPTION
**Pull request by Pangssu**

### CODE REVIEW

**Review by 0xHansLee — APPROVED** [review]
LGTM

File: packages/sdk/src/utils/chain-constants.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import {
 export const DEPOSIT_CONFIRMATION_BLOCKS: {
   [ChainID in L2ChainID]: number
 } = {
-  [L2ChainID.KROMA_SEPOLIA]: 64 as const, // 2 epoch, 1 epoch = 32 slot
+  [L2ChainID.KROMA_SEPOLIA]: 4 as const, // 2 epoch, 1 epoch = 32 slot
```
Comment (chokobole) [inline packages/sdk/src/utils/chain-constants.ts:13]:
```suggestion
  [L2ChainID.KROMA_SEPOLIA]: 4 as const, // 4 slot
```

**Review by seolaoh — APPROVED** [review]
LGTM

**Review by kangsorang — APPROVED** [review]
LGTM

**Comment by Pangssu** [comment]
This PR shouldn't have been merged, but I accidentally merged it. I'll open a new PR to reflect the new deployments and @chokobole's fix.

**Pull request merged**